src/main/interface/settings.c

```diff
@@ -274,6 +274,12 @@ static const char * const lookupTableRatesType[] = {
     "BETAFLIGHT", "RACEFLIGHT"
 };
 
+#ifdef USE_GYRO_EXPERIMENTAL_FILTER
+static const char * const lookupTableStage2FilterType[] = {
+    "NONE", "BQFIR", "FKF"
```

Comment:
Same here, `BIQUAD_RC_FIR2`, the abbreviation/shortening is unnecessary and inaccurate